.migration/docs/setup-guide.md

```diff
@@ -0,0 +1,204 @@
+# S2DM Migration Workflow Setup Guide
+
+This guide explains how to set up the automated S2DM migration workflow in your repository. The workflow automatically generates artifacts (GraphQL schemas, JSON schemas, registry files, and SKOS RDF) when changes are made to your specification files.
+
+## Prerequisites
+
+- A repository with S2DM specification files in a `spec/` directory
+- A `units.yaml` file at the repository root
+- GitHub Actions enabled in your repository
```

Comment:
Add link or instructions on how to do this. The user might not have an idea of it.